<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicInformation</class>
 <widget class="QWidget" name="BasicInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BasicInformation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0" columnminimumwidth="0,0,0,0">
   <item row="1" column="0">
    <widget class="QLabel" name="labelCompoundName_zh">
     <property name="text">
      <string>Compound Name(zh)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditCompoundName_zh"/>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="tBtnName_en_Edit">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QLabel" name="labelState">
     <property name="text">
      <string>State</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QToolButton" name="tBtnCategoryEdit">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelMolecularWeight">
     <property name="text">
      <string>Molecular Weight</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditCompoundName_en"/>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QComboBox" name="comboBoxApplicationArea"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditMolecularWeight"/>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditCompoundType"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelCASNumber">
     <property name="text">
      <string>CAS Number</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditCASNumber"/>
   </item>
   <item row="8" column="3">
    <widget class="QToolButton" name="tBtnApplicationAreaEdit">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QComboBox" name="comboBoxCategory"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelCompoundType">
     <property name="text">
      <string>Compound Type</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLineEdit" name="lineEditStructurePicturePath"/>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QGraphicsView" name="graphicsViewStructure"/>
   </item>
   <item row="8" column="0" rowspan="2">
    <widget class="QLabel" name="labelApplicationArea">
     <property name="text">
      <string>Application Area</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditMolecularFormula"/>
   </item>
   <item row="12" column="2">
    <widget class="QComboBox" name="comboBoxState"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelCompoundName_en">
     <property name="text">
      <string>Compound Name(en)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelMolecularFormula">
     <property name="text">
      <string>Molecular Formula</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="tBtnName_zh_Edit">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QToolButton" name="tBtnUploadStructure">
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditApplicationArea"/>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="labelStructurePicture">
     <property name="text">
      <string>Structure Picture</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditCategory"/>
   </item>
   <item row="6" column="0" rowspan="2">
    <widget class="QLabel" name="labelCategory">
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditCompoundName_en</tabstop>
  <tabstop>tBtnName_en_Edit</tabstop>
  <tabstop>lineEditCompoundName_zh</tabstop>
  <tabstop>tBtnName_zh_Edit</tabstop>
  <tabstop>lineEditMolecularFormula</tabstop>
  <tabstop>lineEditMolecularWeight</tabstop>
  <tabstop>lineEditCASNumber</tabstop>
  <tabstop>lineEditCompoundType</tabstop>
  <tabstop>comboBoxCategory</tabstop>
  <tabstop>tBtnCategoryEdit</tabstop>
  <tabstop>lineEditCategory</tabstop>
  <tabstop>comboBoxApplicationArea</tabstop>
  <tabstop>tBtnApplicationAreaEdit</tabstop>
  <tabstop>lineEditApplicationArea</tabstop>
  <tabstop>graphicsViewStructure</tabstop>
  <tabstop>lineEditStructurePicturePath</tabstop>
  <tabstop>tBtnUploadStructure</tabstop>
  <tabstop>comboBoxState</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
